aws/sagemaker_studio/README.md

```diff
@@ -0,0 +1,45 @@
+# Instructions for Running RAPIDS + SageMaker Studio HPO 
+
+0. Upload train/test data to S3 
+
+   - We offer the dataset for this demo in a public bucket hosted in either the `us-east-1` or `us-west-2` regions:
+   > https://s3.console.aws.amazon.com/s3/buckets/sagemaker-rapids-hpo-us-east-1/   
+   > https://s3.console.aws.amazon.com/s3/buckets/sagemaker-rapids-hpo-us-west-2/
+
+1. Publish a container to the Amazon Elastic Container Registry (ECR)
+
+   - Unfortunately, SageMaker Studio does not provide docker support, however in order to use a custom container, you must first publish it to the ECR. In order to work around this, we recommend running the studio_container_setup.ipynb notebook within a notebook instance. To create a notebook isntance: 
```

Comment:
Did you mean DockerHub? Since technically we are using Docker containers with these examples, meaning it is supported. Also here: https://aws.amazon.com/blogs/machine-learning/bringing-your-own-custom-container-image-to-amazon-sagemaker-studio-notebooks/

Reply:
this meant that we can't work with Docker within a SageMaker Studio notebook, i.e. you can't do "!docker pull ..." from DockerHub or "!docker push ..." to ECR within the SS notebook like you can within regular notebooks. I'll edit that to make it more clear.

the second half of the article "Attaching images using the Studio UI" is describing how you could use the container from Marketplace, which I can also write about but won't be able to test yet since the current 0.17 version is too large to use. 

I can look into following that article tutorial and using the 21.06 version, though! I first just need to make sure I have all of the permissions. I also think it would probably be better as a separate notebook, since the HPO examples can use whatever kernel you want and it's the Estimator object that takes the ECR URI, whereas when you attach images to the Studio domain that image is then used as the kernel for a notebook. so a notebook like that would look more like: https://github.com/sarahyurick/cloud-ml-examples/blob/main/aws/rapids_intro.ipynb where you're able to directly import RAPIDS libraries